.circleci/config.yml

```diff
@@ -62,6 +62,18 @@ jobs:
       # - run:
       #     name: "Run deploy"
       #     command: ./tools/scripts/deploy.sh
+  "Project lint unit tests on Python 3":
+    docker:
+      - image: circleci/python:3.7.4
+    working_directory: ~/test262
+    steps:
+      - checkout
+      - run:
+          name: "Install requirements for linter tool"
+          command: python -m pip install --user --requirement tools/lint/requirements.txt
+      - run:
+          name: "Test the lint tool"
+          command: ./tools/lint/test/run.py
```

Comment:
thanks for adding this!

Reply:
Btw, can we add the `"Project lint unit tests on Python 3"` to the workflows, after line 139 in this file?

I was going to do it in a separate PR but I think it's good to have it going green in this PR as well.